azure-pipelines.yml

```diff
@@ -74,7 +74,7 @@ jobs:
                   JHI_APP: ms-react-gateway-consul-oauth2
                   JHI_ENTITY: sqllight
                   JHI_PROFILE: prod
-                  JHI_E2E: 0
+                  JHI_E2E: 1
```

Comment:
Not sure about this, as I'm pretty sure it still fails because of https://github.com/jhipster/generator-jhipster/issues/12784
it's not related to Cypress, but it's a bug with React when saving entities which have relationship with User

Protactor doesn't detect it as it's less completed than Cypress

Reply:
I would have looked at it, there is a PR to fix it already.
IMO this use case should be tested by sqlfull if not tested yet.
Maybe add a sqlfull to github react+oauth and vue+oauth if missing.

Reply:
yes, this PR should fixed this -> https://github.com/jhipster/generator-jhipster/pull/12787

sqlfull was specially done for default app. I never tested it with OAuth2
But I think we tested too much entities, we should reduce and merge combinations

Reply:
Resolving since it's already enabled at main.